tests/00-script-linter.sh

```diff
@@ -5,23 +5,6 @@ source "${dir}/helpers.bash"
 # dir might have been overwritten by helpers.bash
 dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
-function check_no_sleep {
```

Comment:
I liked this check :(. Can't we keep it?

Reply:
It's just a workaround. It won't be merged. That said, the bash tests are being deprecated.